tools/unicodetools/com/ibm/rbm/resources/RBManager_fi.properties

```diff
@@ -3,7 +3,7 @@
 # @fileLanguage  Finnish
 # @fileCountry   
 # @fileVariant   
-# @fileManager   Jussi Myllym�ki
```

Comment:
There are a large number of diffs in this file which don't show any actual change in the GitHub diff viewer.
Did the encoding of the file accidentally get changed? If so we probably want/need to change it back.

Reply:
Possibly :( for some reason the ä became some weird character in various places at some point, and then i tried to change it back by copying the character again. Thanks for this, let me change it back

Reply:
I may be wrong but this is something I found out:

When I accidentally changed the a with the umlaut (ä), it changed from hex <E4> to <EF><BF><BD>, which I believe is the little question mark in a square.
As far as I understand after some research, <E4> is the hex for ä in UTF-16 LE, and since I saved the file as UTF-8, it may have broken that character. The ä in hex in UTF-8 is <C3> <A4>, which is why git shows the difference. 

Should I change it back to UTF-16? My first guess is that everything should be UTF-8, but since this is a name used in other language/country, maybe UTF-16 is the right thing to do. Please let me know if you thing I should use one specifically :)

Reply:
Hmm, interesting... thanks for the explanation. :)

Doing some [searching](https://stackoverflow.com/a/46926020)... it looks like Java's `ResourceBundle` files are actually encoded using ISO-8859-1, and not UTF-8, nor UTF-16.  At least up until Java 9, where the default was changed to use UTF-8.

I think when you saved the file as UTF-8 it changed the encoding of non-ASCII characters from ISO-8859-1 to UTF-8 -- which is why the file looks like it has so many diffs.

I think we need to change it back to use ISO-8859-1, and _then_ make the changes to fix the typos in the file.

FWIW, if you're using VSCode, you can change the encoding for a file using the status bar at the bottom ([link](https://stackoverflow.com/questions/30082741/change-the-encoding-of-a-file-in-visual-studio-code#:~:text=In%20the%20bottom%20bar%20of%20VSCode%2C%20you%27ll%20see,that%20this%20only%20applies%20to%20newly%20created%20files.)).

For example, here what it looks like with ISO-8859-1 set:
![image](https://user-images.githubusercontent.com/29107334/102543303-f1f35f80-4067-11eb-8b66-a2a4f7b7152f.png)

Reply:
Thank you for this, Jeff! I have changed it back to the correct encoding.

Reply:
Thanks!